src/components/Button.jsx

```diff
@@ -1,16 +1,33 @@
-const Button = ({ children, icon, backgroundColor, textColor, borderColor, fullWidth }) => {
-	return (
-		<button
-			className={`flex justify-center items-center gap-2 px-7 py-4 border font-montserrat text-lg leading-none rounded-full ${
-				backgroundColor ? `${backgroundColor} ${borderColor} ${textColor}` : "bg-coral-red  border-coral-red text-white"
-			}
-			${fullWidth && 'w-full'}
+const Button = ({
+  children,
+  backgroundColor,
+  borderColor,
+  textColor,
+  fullWidth,
+  icon,
+}) => {
+  return (
+    <>
+      <button
+        className={`flex buttonjustify-center items-center gap-2 px-7 py-4 border font-montserrat text-lg leading-none rounded-full hover:bg-white hover:text-coral-red hover:border-coral-red ${
```

Comment:
I think there is a small error in this line, `flex buttonjustify ...`, this is a typo I presume. Please fix this